src/stylesheets/core/placeholders/_table.scss

```diff
@@ -162,13 +162,15 @@ $table-sorted-stripe-text-color: color(
       fill: $table-sorted-header-text-color;
     }
   }
+
   @mixin table-button-sorted-descending-styles {
     @include table-button-default-styles;
     .usa-icon > g.descending {
       fill: $table-sorted-header-text-color;
     }
   }
 
+  /* stylelint-disable selector-class-pattern */
```

Comment:
Added some stylelint disables for come class names that don't pass its BEM test but that we will not change in this release